scripts/new_project.kts

```diff
@@ -248,7 +248,21 @@ object NewProject {
             "$projectPath${fileSeparator}gradlew",
             "-p",
             "$projectPath",
-            "testStagingDebugUnitTest"
+            ":app:testStagingDebugUnitTest"
```

Comment:
then possibly you can use `&` instead of 3 executeCommand?

```suggestion
            ":app:testStagingDebugUnitTest & :data:testDebugUnitTest & :domain:test"
```

Reply:
@sleepylee Thanks for the suggestion, looks a lot better now

Thank you 🎸